### ISSUE TITLE
Can't determine version being used from the UI

### ISSUE THREAD

**Issue by notoriousjere** [johman10/flood-for-transmission#405]
Would you be able to put a version number on the About section of the settings page?
![image](https://user-images.githubusercontent.com/18120535/163923653-280c4461-c7cb-4665-9ede-0aadf57ab41b.png)

---

This issue came to my attention because I am currently using "haugene/docker-transmission-openvpn", which should be pulling the latest version of your alternate UI, per the Dockerfile:
https://github.com/haugene/docker-transmission-openvpn/blob/master/Dockerfile#L9

I've recreated my container and do not see the recent "percent column" update.

**Comment by johman10** [comment]
You're correct, as of right now there is no way of knowing, also because my versioning setup isn't great. If I would want to show it it would require a more solid setup, perhaps something like semantic versioning with proper git commit messages etc. I was already planning on looking into that at some point, but you might have just given me the push to do it rather sooner than later. ;)

Regarding your issue:
> I've recreated my container
Does this mean that you ran a build of the image again? If you just install from DockerHub I think you will have to wait for an update from haugene/docker-transmission-openvpn for the new Flood for Transmission to be included in the docker image. On that same topic there is this discussion ongoing: https://github.com/haugene/docker-transmission-openvpn/discussions/1921

The other way is to go into the running docker container and download and unpack the .zip file manually. (Not sure how restarts of the container will affect this change though).

**Comment by notoriousjere** [comment]
@johman10 

> I was already planning on looking into that at some point, but you might have just given me the push to do it rather sooner than later. ;)

Amazing!

> Does this mean that you ran a build of the image again? If you just install from DockerHub I think you will have to wait for an update from haugene/docker-transmission-openvpn for the new Flood for Transmission to be included in the docker image. On that same topic there is this discussion ongoing: https://github.com/haugene/docker-transmission-openvpn/discussions/1921

Ah crap, yeah I did not run a _rebuild_ -- I am using Portainer and ran a _recreate_ and checked the "pull latest image" toggle. I think I am now realizing after you spelled it out here that the image is 7 days old, but your latest version is 4 days old and that the contents of the Dockerfile doesn't run on a container start -- just once to make the actual image. Kind of a brainfart on my part.

I thought about unpacking the zip manually but i really didn't care enough to do that and surely a container restart would nuke it.
